io.openems.edge.ess.byd.container/src/io/openems/edge/ess/byd/container/EssFeneconBydContainer.java

```diff
@@ -921,17 +942,18 @@ protected ModbusProtocol defineModbus2Protocol() {
 				m(EssFeneconBydContainer.ChannelId.CONTAINER_RUN_NUMBER, new UnsignedWordElement(0x38A9))),
 				new FC16WriteRegistersTask(0x0500,
 						m(EssFeneconBydContainer.ChannelId.SET_SYSTEM_WORKSTATE, new UnsignedWordElement(0x0500)),
-						m(EssFeneconBydContainer.ChannelId.SET_ACTIVE_POWER, new SignedWordElement(0x0501),
-								ElementToChannelConverter.SCALE_FACTOR_3),
-						m(EssFeneconBydContainer.ChannelId.SET_REACTIVE_POWER, new SignedWordElement(0x0502),
-								ElementToChannelConverter.SCALE_FACTOR_3)),
-				new FC16WriteRegistersTask(0x0601, //
-						m(EssFeneconBydContainer.ChannelId.SYSTEM_WORKMODE, new UnsignedWordElement(0x0601))));
+						m(EssFeneconBydContainer.ChannelId.SET_ACTIVE_POWER, new SignedWordElement(0x0501)),
+						m(EssFeneconBydContainer.ChannelId.SET_REACTIVE_POWER, new SignedWordElement(0x0502))));
+		// new FC16WriteRegistersTask(0x0601, //
+		// m(EssFeneconBydContainer.ChannelId.SYSTEM_WORKMODE, new
+		// UnsignedWordElement(0x0601))));
 	}
 
 	@Override
 	public String debugLog() {
-		return "SoC:" + this.getSoc().value().asString() //
+		return "State:" + this.channel(EssFeneconBydContainer.ChannelId.SYSTEM_WORKSTATE).value().asOptionString()
+				+ "|Mode:" + this.channel(EssFeneconBydContainer.ChannelId.SYSTEM_WORKMODE).value().asOptionString()
+				+ "|SoC:" + this.getSoc().value().asString() //
 				+ "|L:" + this.getActivePower().value().asString() //
 				+ "|Allowed:"
 				+ this.channel(ManagedSymmetricEss.ChannelId.ALLOWED_CHARGE_POWER).value().asStringWithoutUnit() + ";"
```

Comment:
Usually we show Charge + Discharge power here: https://github.com/OpenEMS/openems/blob/develop/io.openems.edge.ess.fenecon.commercial40/src/io/openems/edge/ess/fenecon/commercial40/EssFeneconCommercial40Impl.java#L1620